native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/tcp-native/resource-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "resources":{
+    "includes":[{
+      "pattern":"\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
```

Comment:
This resource configuration also may not need to be explicitly added. Can you also check this by removing this file?